server-spi-private/src/main/java/org/keycloak/models/locking/GlobalLockProvider.java

```diff
@@ -17,57 +17,73 @@
 
 package org.keycloak.models.locking;
 
+import org.keycloak.models.KeycloakSessionFactory;
```

Comment:
unused import

Reply:
It is used in Javadoc. Intellij doesn't like it when I remove it, but it is not failing the build, should I remove it?